front/pages/api/w/[wId]/data_sources/[name]/index.ts

```diff
@@ -174,7 +178,7 @@ async function handler(
       if (
         dataSource.connectorId &&
         dataSource.connectorProvider &&
-        !DELETION_ALLOWED_BY_TYPE.includes(dataSource.connectorProvider)
+        !MANAGED_DS_DELETABLE_AS_BUILDER.includes(dataSource.connectorProvider)
```

Comment:
I find the original name to be better than the new one. Why not `DELETION_ALLOWED_BY_CONNECTOR_PROVIDER` ?

Reply:
Because it needs to encode the concpet of builder.